platform-sdk/swirlds-platform-core/src/test/java/com/swirlds/platform/crypto/CryptoArgsProvider.java

```diff
@@ -41,8 +47,7 @@ public class CryptoArgsProvider {
     private static final char[] PASSWORD = "password".toCharArray();
 
     /**
-     * @return 3 sets of arguments, 1 generated, 1 loaded from files, and 1 that generates agreement keys if they are
-     * not loaded from files.
+     * @return 2 sets of arguments, 1 generated, 1 loaded from files.
```

Comment:
That's because the new keys in the .pem files only contain the signing keys. IIRC, the agreement keys are no longer relevant, but if they are needed anywhere, I presume that the key loader will generate them.

So there's only 2 sets of arguments now.